ui/ops/src/dynamic/widgets/graphic/widgets.js

```diff
@@ -25,48 +25,55 @@ export function useWidgetEffects(el, config, sources) {
 
   // allow selecting a child element for the widget to replace
   if (effect.selector) {
-    el = el.querySelector(effect.selector);
+    const els = el.querySelectorAll(effect.selector);
     if (!el) {
```

Comment:
This if is no longer checking what it was before. I think you need something like `if (el.length === 0)`